components/organisms/o-payment.vue

```diff
@@ -279,6 +294,16 @@ export default {
   },
   mounted () {
     createSmoothscroll(document.documentElement.scrollTop || document.body.scrollTop, 0);
+  },
+  computed: {
+    stateOptions () {
+      let countryCode = this.payment.country ? this.payment.country : config.i18n.defaultCountry
+      if (this.states[countryCode] !== null) {
+        return this.states[countryCode]
+      } else {
+        return []
```

Comment:
@AishwaryShrivastav please do the discussed change also here and in the next file

Reply:
Hey @RakowskiPrzemyslaw, I have made those changes here also.